source/framework/vx_graph_verify.c

```diff
@@ -761,8 +761,8 @@ static vx_status ownGraphCalcInAndOutNodes(vx_graph graph)
                 {
                     node_next = graph->nodes[node_next_idx];
                     for (prm_next_idx = 0;
-                         prm_next_idx < ownNodeGetNumParameters(node_next) &&
-                           ((vx_status)VX_SUCCESS == status);
+                         prm_next_idx < (ownNodeGetNumParameters(node_next) &&
```

Comment:
This placement of parenthesis doesn't look right to me.  I would have expected it to be

(prm_next_idx < ownNodeGetNumParameters(node_next)) &&
                           ((vx_status)VX_SUCCESS == status);

Reply:
oh, you're totally right, thank you for reviewing, this is now fixed.